src/Tests/dotnet.Tests/dotnet-msbuild/GivenDotnetBuildInvocation.cs

```diff
@@ -56,7 +56,7 @@ public void MsbuildInvocationIsCorrect(string[] args, string expectedAdditionalA
 
         [Theory]
         [InlineData(new string[] { "-f", "tfm" }, "-target:Restore", "-property:TargetFramework=tfm")]
-        [InlineData(new string[] { "-p:TargetFramework=tfm" }, "-target:Restore", "-p:TargetFramework=tfm")]
+        [InlineData(new string[] { "-p:TargetFramework=tfm" }, "-target:Restore", "-property:TargetFramework=tfm")]
```

Comment:
since all `-p` forms are normalized to `-property`, this needed updating as well.